src/test/Makefile

```diff
@@ -39,9 +43,9 @@ atomic_unittest_SRC := \
 		$(USER_DIR)/build/atomic.c \
 		$(TEST_DIR)/atomic_unittest_c.c
 
-baro_bmp085_unittest_SRC := \
-		$(USER_DIR)/drivers/barometer/barometer_bmp085.c \
-		$(USER_DIR)/drivers/io.c
+#baro_bmp085_unittest_SRC := \
```

Comment:
Why was this commented out? This has to be reverted.

Reply:
This was commented out because the test had not been used at all: It [is called `.cc.txt`](https://github.com/betaflight/betaflight/blob/master/src/test/unit/baro_bmp085_unittest.cc.txt), not `.cc`, and therefore it never executes. I did not investigate why this test is disabled and took it as a fact.

The key reason why I commented out all unused tests was [this new consistency check](https://github.com/betaflight/betaflight/blob/2eb6aa27d486624d7d18232ac25d46d346ca7900/src/test/Makefile#L579-L582) that I added. It checks

  * whether each `test_SRC` variable has a `test.cc` file. [not fulfilled in this case]
  * whether each `test.cc` file has a `test_SRC` variable.

So, would you like me to try to enable the test by first renaming `baro_bmp085_unittest.cc.txt` to `baro_bmp085_unittest.cc`? I can give that a try, if the test works.

Reply:
Investigated this one: It's missing `GPIOA_BASE`, a symbol that's currently accessed in `io.c`, but probably not defined anywhere except `SITL`'s `target.h`.

That^^^ was likely the reason why someone renamed the test to `.txt`.

Reply:
See comment below. Commented out code should always have an explanation as to why it was commented out.